templates/etc/httpd/conf.d/pulp_rpm.conf.erb

```diff
@@ -2,17 +2,32 @@
 # Apache configuration file for pulp web services and repositories
 #
 
-# -- HTTPS Repositories ---------
-Alias /pulp/repos /var/www/pub/yum/https/repos
+AddType application/x-pkcs7-crl .crl
+AddType application/x-x509-ca-cert .crt
```

Comment:
I don't know of anyone who's downloading CAs or CRLs from this location but a Pulp person would know for sure.

It doesn't hurt anything to have this in here so I say just leave it :hamster: